src/test/java/br/com/caelum/comidinhas/tipoCozinha/TipoCozinhaControllerTest.java

```diff
@@ -0,0 +1,123 @@
+package br.com.caelum.comidinhas.tipoCozinha;
+
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.util.*;
+
+import static br.com.caelum.comidinhas.utils.SerializadorJson.paraJson;
+import static org.hamcrest.Matchers.is;
+import static org.mockito.Mockito.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest
+@AutoConfigureMockMvc
```

Comment:
Dá pra trocar todas essas anotações por apenas:

```suggestion
@WebMvcTest(TipoCozinhaController.class)
```

Um `@WebMvcTest` sobe o Controller, o Validator, mas não precisa de conexão com o BD.